portals/publisher/source/src/app/components/Apis/Details/Documents/View.jsx

```diff
@@ -119,6 +120,20 @@ function View(props) {
                 const { body } = doc;
                 setDoc(body);
                 if (body.sourceType === 'MARKDOWN' || body.sourceType === 'INLINE') loadContentForDoc();
+
+                if (body.sourceType === 'FILE') {
+                    const promised_get_content = restAPI.getFileForDocument(api.id, documentId);
+                    promised_get_content
+                        .then((done) => {
+                            setIsFileAvailable(true);
+                        })
+                        .catch((error) => {
+                            if (process.env.NODE_ENV !== 'production') {
```

Comment:
We don't need this check here, and for error logs let's use 

console.error(error)

Reply:
fixed with https://github.com/wso2/apim-apps/pull/77/commits/1432da413016cdc640c7127513630b89f9a96a87